projects/rero/ng-core/src/lib/record/search/record-search-page.component.ts

```diff
@@ -134,11 +134,11 @@ export class RecordSearchComponent implements OnInit {
       this.types = data.types;
     }
 
-    if (typeof data.showSearchInput !== 'undefined') {
+    if (data.showSearchInput != null) {
```

Comment:
`!== null`?

Reply:
!= null checks both for undefined and null, I think it's better in this case.